client/my-sites/media-library/test/list-item-image.jsx

```diff
@@ -29,7 +29,8 @@ describe( 'MediaLibraryListItem image', () => {
 		}
 	} );
 
-	const getPhotonUrl = () => photon( fixtures.media[ 0 ].URL, { resize: `${ WIDTH },${ WIDTH }` } );
+	const getResizedUrl = () =>
```

Comment:
Why is this method duplicated? I'm pretty sure just the original `getResizedUrl` needs to be there. It's the reason behind the linting failure.

Reply:
Speaking of linting, there are a couple of [errors](https://circleci.com/gh/Automattic/wp-calypso/204662?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link):

```
line 131, col 4, Error - Visible, non-interactive elements with click handlers must have at least one keyboard listener. (jsx-a11y/click-events-have-key-events)

line 131, col 4, Error - Static HTML elements with event handlers require a role. (jsx-a11y/no-static-element-interactions)

```

All these accessibility issues need to be addressed at some point, but I don't think it's in scope for this PR.

You can disable these warning by following this [example](https://github.com/Automattic/wp-calypso/blob/87ea7231a319c3b9a06fb021308324ed66b4db23/client/components/title-format-editor/token.jsx#L6).

Hope that helps!

Reply:
I looked at this today but I'm not sure I understand. I'm only seeing a single definition for getResizedUrl() then it's called a few times later. I'm also interested in solving the linting issues but not entirely sure how.

Sorry, I know this isn't all that important. Just unsure how to proceed 😢

Reply:
No worries @rfaile313 

I'll take a quick look and push a commit if that's okay to you. I'll write a couple of notes to explain the changes where appropriate.

Reply:
Just to clarify, here's what I'm seeing in the [test file](https://github.com/Automattic/wp-calypso/blob/e342f7c0f08ffdd3a2aae8e1b114a900f32c5cf2/client/my-sites/media-library/test/list-item-image.jsx#L32).

```
	const getResizedUrl = () =>
		photon( fixtures.media[ 0 ].URL, { resize: `${ WIDTH },${ WIDTH }` } );
	const getResizedUrl = () =>
		resize( fixtures.media[ 0 ].URL, { resize: `${ WIDTH },${ WIDTH }` } );
``

There are two functions with the same name. The one calling `photon()` has been added in e342f7c0f08ffdd3a2aae8e1b114a900f32c5cf2. We'll ditch it.

As for the other linting errors, wrapping the rendered element in the `eslint-disable` comments will ensure that the linter doesn't complain. For example, to address the linting errors in [my-sites/media-library/list-item.jsx](https://github.com/Automattic/wp-calypso/blob/e342f7c0f08ffdd3a2aae8e1b114a900f32c5cf2/client/my-sites/media-library/list-item.jsx#L131):


	/* eslint-disable jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions */
	<div className={ classes } style={ style } onClick={ this.clickItem } { ...props }>
		...
	</div>
	/* eslint-enable jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions */

I'll rebase and make these changes. 👍

Reply:
Hey Thanks! 

Am I crazy? 

In my local commit and in the github PR here 

>const getResizedUrl = () =>
		photon( fixtures.media[ 0 ].URL, { resize: `${ WIDTH },${ WIDTH }` } );
	const getResizedUrl = () =>
		resize( fixtures.media[ 0 ].URL, { resize: `${ WIDTH },${ WIDTH }` } );

I had removed the the first instance (calling photon) and I wasn't seeing it in the code. I must be losing it 😅

Thanks for your patience with this. I'll make these changes as well on my local branch and commit but if you have it from here that's cool too. Appreciate your time with this. A12 devs rock!